### ISSUE TITLE
Gedit 47.0 compatibility

### ISSUE THREAD

**Issue by yochananmarqos** [jefferyto/gedit-ex-mortis#11]
The plugin no longer works as of 47.0.

```
Traceback (most recent call last):
  File "/home/yochanan/.local/share/gedit/plugins/ex-mortis/__init__.py", line 330, in on_app_window_added
    self.setup_window(window)
  File "/home/yochanan/.local/share/gedit/plugins/ex-mortis/__init__.py", line 224, in setup_window
    window_manager.track_window(window)
  File "/home/yochanan/.local/share/gedit/plugins/ex-mortis/windowmanager.py", line 103, in track_window
    connect_handlers(
  File "/home/yochanan/.local/share/gedit/plugins/ex-mortis/utils/__init__.py", line 61, in connect_handlers
    handler_ids.append(target.connect(signal, fn, *args))
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by jefferyto** [comment]
Thanks for continuing to let me know of these incompatibilities - this should be fixed in the new version (0.2.3).

**Comment by yochananmarqos** [comment]
Well, thanks for creating this extension!

By the way, for now I'm sticking with Gedit 46.2 until 48.0 is released as 47.0 is broken. See my [upstream issue](https://gitlab.gnome.org/GNOME/gedit/-/issues/590).
